draft-ietf-scitt-architecture.md

```diff
@@ -175,6 +177,158 @@ SCITT achieves this by having producers publish information in a Transparency Se
+     |           |
+     +-----+-----+
+           |
+     +-----+-----+
+     |           |        Malicious plug-ins;
+     |  Commit   |        Malcious commit
+     |           |
+     +-----+-----+
+           |
+     +-----+-----+
+     |           |        Modify build tasks or build environment;
+     |   Build   |        Poison build agent/compiler;
+     |           |        Tamper with build cache
+     +-----+-----+
+           |
+     +-----+-----+
+     |           |        Compromise test tools;
+     |    Test   |        Falsification of test results
+     |           |
+     +-----+-----+
+           |
+     +-----+-----+
+     |           |        Use bad package;
+     |  Package  |        Compromise package repository
+     |           |
+     +-----+-----+
+           |
+     +-----+-----+
+     |           |        Modify release tasks;
+     |  Release  |        Modify build drop prior to release
+     |           |
+     +-----+-----+
+           |
+     +-----+-----+
+     |           |
+     |  Deploy   |        Tamper with versioning and update process
+     |           |
+     +-----------+
+~~~
+{: #lifecycle-threats title="Example SSC Life-Cycle Threats"}
+
+DevSecOps often depends on third-party and open-source software.
+These dependencies can be quite complex throughout the supply chain and render the checking of lifecycle compliance difficult.
+There is a need for manageable auditability and accountability of digital products.
+Typically, the range of types of statements about digital products (and their dependencies) is vast, heterogeneous, and can differ between community policy requirements.
+Taking the type and structure of all statements about digital and products into account might not be possible.
+Examples of statements may include commit signatures, build environment and parameters, software bill of materials, static and dynamic application security testing results, fuzz testing results, release approvals, deployment records, vulnerability scan results, and patch logs.
+In consequence, instead of trying to understand and describe the detailed syntax and semantics of every type of statement about digital products, the SCITT architecture focuses on ensuring statement authenticity, visibility/transparency, and intends to provide scalable accessibility.
+The following use cases illustrate the scope of SCITT and elaborate on the generic problem statement above.
+
+## Eclectic SSC Use Cases
+
+The three following use cases are a specialization derived from the generic problem statement above.
+These three use case and more were incubated and selected from this Internet-Draft: {{-all-scitt-use-cases}}.
+
+### Security Analysis of a Software Product
+
+A released software product is often accompanied by a set of complementary statements about it's security compliance.
+This gives enough confidence to both producers and consumers that the released software has a good security standard and is suitable to use.
```

Comment:
```suggestion
This gives enough confidence to both producers and consumers that the released software meets expected security standards and is suitable to use.
```

Probably best to think in terms of expectations, rather than good/bad.